ConfigureFreedns.sh

```diff
@@ -201,10 +201,6 @@ do
          sleep 60
     else
         # worked, geting out of the loop.
-        # Add log
```

Comment:
The ConfigureFreedns.sh script is only called by Install Nightscout Phase 2.  The Install Nightscout phase 2 already creates a log.
Having both of them create a log is not needed any longer.  We needed that because we also ran ConfigureFreedns alone when people used noip dot com.   But, we don't support that any longer.
That's why we removed it from the menu.  But, when I did that, I forgot to remove its log.  I am removing it here as something that should have been done before.